### PR TITLE
[PWN-4640] Add intercom in the flow of creating and restoring

### PR DESCRIPTION
**Pull request by qwerkly**
## Link jira to issue
https://p2pvalidator.atlassian.net/browse/PWN-4640

## Description of the changes
-

### CODE REVIEW

File: p2p_wallet/Scenes/CreateOrRestoreWallet/CreateWallet/Steps/EnterPhoneNumber/EnterPhoneNumberViewModel.swift [hunk 1]
```diff
@@ -26,6 +26,7 @@ final class EnterPhoneNumberViewModel: BaseOTPViewModel {
     @Injected private var reachability: Reachability
     @Injected private var notificationService: NotificationService
     @Injected private var countriesAPI: CountriesAPI
+    @Injected private var helpCenterLauncher: HelpCenterLauncher
```
Comment (lisemyon) [inline p2p_wallet/Scenes/CreateOrRestoreWallet/CreateWallet/Steps/EnterPhoneNumber/EnterPhoneNumberViewModel.swift:29]: RestoreCustomDelegatedCoordinator has openHelp() method, can you look at it? And all "openInfo" methods in RestoreCustomDelegatedCoordinator, RestoreSocialDelegatedCoordinator, RestoreWalletCoordinator, BindingPhoneNumberDelegatedCoordinator should open intercom I guess. Should they be deleted or be written somehow?

Comment (bigearsenal) [reply p2p_wallet/Scenes/CreateOrRestoreWallet/CreateWallet/Steps/EnterPhoneNumber/EnterPhoneNumberViewModel.swift:29]: @qwerkly Can you answer to @lisemyon's question?

**Review by bigearsenal — APPROVED** [review]

File: p2p_wallet/Scenes/CreateOrRestoreWallet/RestoreWallet/Steps/RestoreCustom/RestoreCustomDelegateCoordinator.swift [hunk 2]
```diff
@@ -52,7 +55,6 @@ final class RestoreCustomDelegatedCoordinator: DelegatedCoordinator<RestoreCusto
 
     private func openHelp() {
         openInfo()
```
Comment (lisemyon) [inline p2p_wallet/Scenes/CreateOrRestoreWallet/RestoreWallet/Steps/RestoreCustom/RestoreCustomDelegateCoordinator.swift:57]: helpLauncher.launch()

Comment (lisemyon) [reply p2p_wallet/Scenes/CreateOrRestoreWallet/RestoreWallet/Steps/RestoreCustom/RestoreCustomDelegateCoordinator.swift:57]: openInfo - openHelp

Comment (bigearsenal) [reply p2p_wallet/Scenes/CreateOrRestoreWallet/RestoreWallet/Steps/RestoreCustom/RestoreCustomDelegateCoordinator.swift:57]: @lisemyon what do you mean?

Comment (lisemyon) [reply p2p_wallet/Scenes/CreateOrRestoreWallet/RestoreWallet/Steps/RestoreCustom/RestoreCustomDelegateCoordinator.swift:57]: we made a call a discussed this method. it should open help but now it opens terms. we looked through all screens and there are several places which requires intercom also

**Review by TrGiLong — APPROVED** [review]

**Review by lisemyon — APPROVED** [review]

**Pull request merged**